packages/fuels-macros/src/abigen/code_gen/custom_types.rs

```diff
@@ -0,0 +1,741 @@
+use std::collections::HashSet;
```

Comment:
This file was moved, but GitHub didn't pick up on that. 

You'll see it later on in red.